src/test/java/io/adobe/cloudmanager/AdobeClientCredentialsTest.java

```diff
@@ -0,0 +1,49 @@
+package io.adobe.cloudmanager;
+
+/*-
+ * #%L
+ * Adobe Cloud Manager Client Library
+ * %%
+ * Copyright (C) 2020 Adobe Inc.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.KeySpec;
+import java.security.spec.PKCS8EncodedKeySpec;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class AdobeClientCredentialsTest {
+
+    @Test
+    public void testGetKeyFromPem() throws IOException, NoSuchAlgorithmException, InvalidKeySpecException {
+        String pem = IOUtils.resourceToString("/keys/private-key-pkcs8-rsa.pem", StandardCharsets.US_ASCII);
```

Comment:
Please generate the key on the fly, or GitHub will raise an issue that private keys are part the code base. Here's an example in this project:

https://github.com/adobe/aio-lib-java-cloudmanager/blob/1515519af147a005727ff4bd72ac2499404ceca4/src/test/java/io/adobe/cloudmanager/impl/IdentityManagementApiImplTest.java#L68-L72

Reply:
IMHO there is no Java API to serialize a PrivateKey into PEM PKCS#8 format.
There must be a way to disable the warning in GitHub as this private key does not grant access to anything....